src/minimax_server/core.clj

```diff
@@ -1 +1,10 @@
-(ns minimax-server.core)
+(ns minimax-server.core
+  (:import [scarvill.httpserver.server Server])
+  (:require [minimax-server.computer-player :as computer]
+            [minimax-server.server-configuration :refer [minimax-server-config]]))
+
+(def ttt-server
+  (new Server (minimax-server-config 5000 computer/stupid-move)))
```

Comment:
Definitely, Aaron mentioned to us that we should have separate endpoints for the stupid player and the unbeatable player. Spencer made another story for this.